### PR TITLE
Secondattemt

### PR DESCRIPTION
**Pull request by Snurre1**
fixed the variables.scss, it will link now. added primary color and light color. 
added Edit buttons on the post page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,165 @@
 {
-  "name": "ca-project",
+  "name": "social-media-app",
   "version": "0.0.1",
-  "description": "project for school",
+  "description": "school project",
   "main": "index.js",
+  "dependencies": {
+    "accepts": "^1.3.8",
```
Comment (Fermain) [inline package.json]: Too many dependencies, you should only have bootstrap in here. This usually happens when you run `npm init` on an existing project. Simply delete all of the items in dependencies and `npm i bootstrap` again.

File: profile.html [hunk 2]
```diff
@@ -69,30 +69,30 @@ <h1>My profile</h1>
...
                 <p class="h6">Thomas</p>
               </div>
-              <div class="col-sm-9 p-3 border">
+              <div class="col-sm-9 p-3">
                 <p>My post: WE GOT THIS!</p>
               </div>
             </div>
-            <div class="container border border-secondary border-1">
+            <div class="container border border-secondary border-1 bg-light">
              <div class="col-3 p-3">
```
Comment (Fermain) [inline profile.html]: Columns must be nested in rows:

```html
<div class="container">
  <div class="row">
    <div class="col-">
```

**Review by Fermain — CHANGES_REQUESTED** [review]
It's better but there are still improvements to be made with row/col usage

File: package.json (hunk 1 above)
Comment (Snurre1) [reply package.json]: fixed that, but nothing came back when I did `` npm i boostrap``

File: profile.html (hunk 2 above)
Comment (Snurre1) [reply profile.html]: fixed

File: package.json (hunk 1 above)
Comment (Fermain) [reply package.json]: That's because you installed as a devDependency, you can see it there.

```bash
npm uninstall bootstrap
npm install bootstrap
```

Comment (Snurre1) [reply package.json]: its in the dependencies now

File: package.json [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
+  "name": "social-media-app",
   "version": "0.0.1",
-  "description": "project for school",
+  "description": "school project",
   "main": "index.js",
+   "dependencies": {
+    "boostrap": "^2.0.0"
+  },
+  "devDependencies": {
+    "bootstrap": "^5.2.1",
```
Comment (Fermain) [inline package.json]:
```suggestion
```

Comment (Fermain) [reply package.json]: You should only have it in dependencies, not twice

Comment (Snurre1) [reply package.json]: done now?

**Review by Fermain — APPROVED** [review]

**Pull request merged**